velox/expression/tests/ExpressionRunnerTest.cpp

```diff
@@ -52,6 +53,12 @@ DEFINE_string(
     "flag are mutually exclusive. If both are specified, --sql is used and "
     "--sql_path is ignored.");
 
+DEFINE_string(
+    engine,
+    "presto",
+    "Engine to use for expression evaluation. Currently supported values are "
```

Comment:
Not sure if 'engine' is the right word, since all we are testing is the registry of functions. Maybe rename it to 'registry' ?

Reply:
Okay, I will that in the next batch of changes.